src/Illuminate/Support/HtmlString.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Illuminate\Support;
+
+use Illuminate\Contracts\Support\Htmlable;
+
+class HtmlString implements Htmlable
```

Comment:
What about the whole idea? I not too keen myself tbhm but I believe it'll scratch an itch for a lot of people.